Docker/pgbouncer/pgbouncer.ini

```diff
@@ -0,0 +1,12 @@
+[databases]
+adaguc = host=adaguc-db port=5432 dbname=adaguc
```

Comment:
Maybe good to add a comments that this gets overwritten in production.

Maybe for all lines that get changes, also userlist.txt.

Reply:
I've added comments :+1: